docs/TimerMocks.md

```diff
@@ -167,6 +167,28 @@ it('calls the callback after 1 second via advanceTimersByTime', () => {
 
 Lastly, it may occasionally be useful in some tests to be able to clear all of the pending timers. For this, we have `jest.clearAllTimers()`.
 
+## Advance Timers to next Frame
+
+In applications, often you want to schedule work inside of an animation frame (via `requestAnimationFrame`). We expose a convenance method `jest.runToFrame()` to advance all timers enough milliseconds to execute all actively scheduled animation frames.
```

Comment:
Should we say that it's not always 1000/60 ms, but some math from where the clock has already been advanced to?

Reply:
I can try to make that clearer if you think more clarity is needed. I thought that "advance all timers enough milliseconds" explained the intention - but I can make it more explicit

Reply:
Yeah, it's probably fine 🙂 

Just noticed a typo when reading through again - "convenance"

Reply:
Cheers